sig-docs/annual-report-2021.md

```diff
@@ -0,0 +1,98 @@
+    Are you keeping recordings up to date/trends in community members watching recordings?**
+    
+    Quite small (usually 5-10 people) and generally we try to make people feel comfortable to give their opinion.
+    Meeting notes are up to date. We have recording of the meetings uploaded.
+
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired?
+Are OWNERS files up to date in these areas?**
+
+<!-- TODO -->
+
+**How does the group get updates, reports, or feedback from Working Groups? Are there any springing up or being retired? 
+Are OWNERS files up to date in these areas?**
+
+<!-- TODO -->
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+<!-- TODO -->
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+<!-- TODO -->
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+We don't have an official way of measuring membership.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+PR are generally open for anyone to review. We have [PR wrangler rotation](https://github.com/kubernetes/website/wiki/PR-Wranglers).
+In general each of the approver will taking the rotation. Under exceptional circumstances where there's lots of
+pressure for the wrangler, another approver usually will be willing to help.
+
+For technical reviews, other than SIG Docs Tech Leads, we also notify respective SIGs with necessary knowledge of the
+technical aspect of the content.
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? 
+What are some activities that the group does to encourage this? 
+What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+There is no official onboarding process.
+
+**What programs do you participate in for new contributors?**
+
+<!-- TODO -->
+
+**Does the group have contributors from multiple companies/affiliations? 
+Can end users/companies contribute in some way that they currently are not?**
+
+<!-- TODO -->
+
+## Current initiatives and project health
+
+[ ] Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members.
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? 
+Currently underway? What are some of the longer tail projects that your group is working on?**
+
+<!-- TODO -->
+
+**Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?**
+
+<!-- TODO -->
+
+**What areas and/or subprojects does the group need the most help with?**
+
+<!-- TODO -->
+
+**What's the average open days of a PR and Issue in your group? / what metrics does your group care about and/or measure?**
+
+<!-- TODO -->
```

Comment:
Do we have something that can be used to track how many Issues and PR that available in general per day?

Reply:
https://k8s.devstats.cncf.io/ is useful here
We usually look at total open PRs, total open issues (both for k/website specifically), and total open PRs labelled “language/en”.

Reply:
Also see https://k8s.devstats.cncf.io/d/25/open-pr-age-by-repository-group?orgId=1&var-period=d7&var-repogroup_name=SIG%20Docs&var-kind_name=All

Reply:
I put the graph and the link to the graph, PTAL